### ISSUE TITLE
rhai fmt should honor tabsize

### ISSUE THREAD

**Issue by schungx** [rhaiscript/lsp#101]
The formatter should honor the default Rhai tab-size:

```json
[rhai]": {
    "editor.tabSize": 4
}
```

unless explicitly overwritten, I suppose.

Right now it auto-formats to 2 spaces.

**Comment by tamasfe** [comment]
Weird, if you change the tabs for a file, it does. I guess we'll have to manually read this config in the language server.

**Comment by schungx** [comment]
That's correct.  I can manually change the tab setting for an individual file and `rhai fmt` will honor it.  However, if I don't do that, it somehow defaults to 2 spaces.
